code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/URLUtilsTests.java

```diff
@@ -0,0 +1,31 @@
+/*
+  Copyright 2022 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.edge.identity;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class URLUtilsTests {
+
+	@Test
+	public void test_generateURLVariablesPayload_emptyValuesPassed_returnsStringWithURLPrefixOnly() {
+		String actual = URLUtils.generateURLVariablesPayload("", "", "");
+		assertEquals("adobe_mc=", actual);
```

Comment:
As I mentioned in another comment, we shouldn't return a parameter without a value. This result should be "adobe_mc=null" or something similar.